alibi/explainers/similarity/grad.py

```diff
@@ -23,28 +22,18 @@
     import torch
 
 
-class Task(Enum):
+def get_options_string(enum) -> str:
```

Comment:
`enum: Enum` ?

Reply:
`"' | '".join(enum)` expects type `Iterable[str]` so complains about getting `Enum` instead. Should I ignore the mypy issue explicitly? Or is there a better way of managing this?

Reply:
Ah I see. Probably the proper way of doing this would be define a `StrEnum(str, Enum)` class and make all enum's inherit from that, but let's not complicate, `enum: str` should be enough.